lib/chat_api/customers.ex

```diff
@@ -196,15 +196,18 @@ defmodule ChatApi.Customers do
     end
   end
 
-  @spec find_by_email(binary() | nil, binary()) :: Customer.t() | nil
-  def find_by_email(nil, _account_id), do: nil
+  @spec find_by_email(binary() | nil, binary(), atom() | list(atom()) | keyword()) ::
+          Customer.t() | nil
+  def find_by_email(email, account_id, preloads \\ [])
+  def find_by_email(nil, _account_id, _preloads), do: nil
 
-  def find_by_email(email, account_id) do
+  def find_by_email(email, account_id, preloads) do
```

Comment:
Used in this commit to preload messages in tests, but I imagine that this could be useful in normal application code as well.